descriptors/analysis_engines/demo.xml

```diff
@@ -133,22 +133,25 @@
         <name>camera_config_files</name>
         <value>
           <array>
-            <string>config_kinect_robot.ini</string>
+            <string>config_tango.ini</string>
```

Comment:
these changes should not be in the pull-request. Changes to demo.xml would break the online tutorial!